docs/infrastructure/deployment-targets/reflected-targets/index.md

```diff
@@ -0,0 +1,88 @@
+1. Click **SAVE**.
+
+## Enable deployments without a target
+
+Using automatic discovery of targets during means that there may be no existing targets at the start of a deployment. The deployment setting allowing deployments without a target is enabled by default for new projects, or it can be enabled or disabled for existing projects.
+
+1. Navigate to **{{Projects,{Project name},Deployments,Settings}}**.
+1. Expand the **Deployment Targets Required** and select the "Allow deployments to be created when there are no deployment targets" option.
+1. Click **SAVE**.
+
+## Configure credentials for discovery
+
+To perform discovery of targets from cloud resources, Octopus uses a well-known set of variables from within your project to provide the credentials for authentication to the cloud provider. Variables can be [scoped](/docs/projects/variables/index.md#scoping-variables) allowing for different credentials to be used when performing discovery as required (for example in different environments).
+
+The variables used are different for each supported cloud provider.
+
+### Azure
+
+To discover Azure cloud resources, Octopus uses an [Azure account](/docs/projects/variables/azure-account-variables.md) variable named **Octopus.Azure.Account**.
+
+### AWS
+
+To discover AWS cloud resources, Octopus uses an [AWS account](/docs/projects/variables/aws-account-variables.md) variable named **Octopus.AWS.Account**. TODO: Additional variables ??? can be used to authenticate using an EC2 IAM role.
+
+## Tag cloud resources
+
+Octopus looks for tags applied to cloud resources to discover and create deployment targets automatically for you.
+
+Tags are in the format `octopus-{scope}` and support the following for discovery. Note that only a single value is supported in tags at the moment.
+
+- `octopus-environment`: The name of the [environment](/docs/infrastructure/environments) the target can be used in during deployments. This tag is required and only deployments for the environment will discover the target.
+- `octopus-role`: The [role](/docs/infrastructure/deployment-targets#target-roles) that should be applied to the target. This tag is required and only deployments with a step that matches the role will discover the target.
+- `octopus-space`: The name of the [space](/docs/administration/spaces) the target can be used in. This tag is optional and if present only deployments within the matching space can discover the target.
+- `octopus-project`: The name of the [project](/docs/projects) the target will be discovered by. This tag is optional and if present only deployments for the matching project can discover the target.
+- `octopus-tenant`: The name of the [tenant](/docs/projects) the target can be discovered for. This tag is optional and if present only deployments for the matching tenant will discover the target.
+
+## Examples
+
+### ECS Cluster
```

Comment:
I added an ECS cluster example here instead of an Azure one as feels like we don't want to appear to be too Azurey/Microsofty. The first piece of work is for reflected Azure Web App targets so I suppose we could add that first and then add an ECS example afterwards when that is available.